CI failure unrelated 
```
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.8.0-jboss-2:compile (default-compile) on project quarkus-integration-test-hibernate-orm-panache: Compilation failure: Compilation failure: 
[ERROR] /home/vsts/work/1/s/integration-tests/hibernate-orm-panache/src/main/java/io/quarkus/it/panache/TestEndpoint.java:[91,51] cannot find symbol
[ERROR]   symbol:   method withLock(javax.persistence.LockModeType)
[ERROR]   location: interface io.quarkus.hibernate.orm.panache.PanacheQuery<io.quarkus.hibernate.orm.panache.PanacheEntityBase>
[ERROR] /home/vsts/work/1/s/integration-tests/hibernate-orm-panache/src/main/java/io/quarkus/it/panache/TestEndpoint.java:[141,22] method findById in class io.quarkus.hibernate.orm.panache.PanacheEntityBase cannot be applied to given types;
[ERROR]   required: java.lang.Object
[ERROR]   found: java.lang.Long,javax.persistence.LockModeType
[ERROR]   reason: cannot infer type-variable(s) T
[ERROR]     (actual and formal argument lists differ in length)
[ERROR] /home/vsts/work/1/s/integration-tests/hibernate-orm-panache/src/main/java/io/quarkus/it/panache/TestEndpoint.java:[371,54] cannot find symbol
[ERROR]   symbol:   method withLock(javax.persistence.LockModeType)
[ERROR]   location: interface io.quarkus.hibernate.orm.panache.PanacheQuery<io.quarkus.it.panache.Person>
[ERROR] /home/vsts/work/1/s/integration-tests/hibernate-orm-panache/src/main/java/io/quarkus/it/panache/TestEndpoint.java:[420,25] method findById in interface io.quarkus.hibernate.orm.panache.PanacheRepositoryBase<Entity,Id> cannot be applied to given types;
[ERROR]   required: java.lang.Long
[ERROR]   found: java.lang.Long,javax.persistence.LockModeType
[ERROR]   reason: actual and formal argument lists differ in length
[ERROR] -> [Help 1]
```

--- a/extensions/vertx/deployment/src/main/java/io/quarkus/vertx/deployment/VertxProcessor.java
+++ b/extensions/vertx/deployment/src/main/java/io/quarkus/vertx/deployment/VertxProcessor.java
@@ -25,6 +25,7 @@ import io.quarkus.arc.processor.AnnotationStore;
 import io.quarkus.arc.processor.AnnotationsTransformer;
 import io.quarkus.arc.processor.BeanInfo;
 import io.quarkus.arc.processor.BuildExtension;
+import io.quarkus.deployment.Capabilities;
 import io.quarkus.deployment.GizmoAdaptor;
 import io.quarkus.deployment.annotations.BuildProducer;
 import io.quarkus.deployment.annotations.BuildStep;
@@ -51,6 +52,11 @@ class VertxProcessor {
     @Inject
     BuildProducer<ReflectiveClassBuildItem> reflectiveClass;
 
+    @BuildStep(providesCapabilities = Capabilities.RESTEASY_JSON_EXTENSION)
+    FeatureBuildItem feature() {
+        return new FeatureBuildItem(FeatureBuildItem.VERTX);
+    }
+
     @BuildStep
     AdditionalBeanBuildItem registerBean() {
         return AdditionalBeanBuildItem.unremovableOf(VertxProducer.class);
@@ -59,13 +65,11 @@ class VertxProcessor {
     @BuildStep
     @Record(ExecutionTime.RUNTIME_INIT)
     VertxBuildItem build(CoreVertxBuildItem internalVertx, VertxRecorder recorder, BeanContainerBuildItem beanContainer,
-            BuildProducer<FeatureBuildItem> feature,
             List<EventConsumerBusinessMethodItem> messageConsumerBusinessMethods,
             BuildProducer<GeneratedClassBuildItem> generatedClass,
             AnnotationProxyBuildItem annotationProxy, LaunchModeBuildItem launchMode, ShutdownContextBuildItem shutdown,
             BuildProducer<ServiceStartBuildItem> serviceStart,
             List<MessageCodecBuildItem> codecs, RecorderContext recorderContext) {
-        feature.produce(new FeatureBuildItem(FeatureBuildItem.VERTX));
         Map<String, ConsumeEvent> messageConsumerConfigurations = new HashMap<>();
         ClassOutput classOutput = new GizmoAdaptor(generatedClass, true);
         for (EventConsumerBusinessMethodItem businessMethod : messageConsumerBusinessMethods) {

